bucket/v2rayn.json

```diff
@@ -24,6 +30,15 @@
         "            New-Item -ItemType HardLink -Path \"$dir\" -Name $name -Target \"$(appdir v2ray $global)\\current\\$name\" | Out-Null",
         "        }",
         "    }",
+        "}",
+        "foreach ($form in @('*.exe')) {",
```

Comment:
Same as previous comment.

Reply:
The executable file name from xray is different from the one from v2ray.